integrations/domain-mapping/domain-mapping.php

```diff
@@ -17,7 +17,7 @@ class PLL_Domain_Mapping {
 	 */
 	public function __construct() {
 		if ( function_exists( 'redirect_to_mapped_domain' ) ) {
-			if ( ! defined( 'PLL_CACHE_HOME_URL' ) && ( $options = get_option( 'polylang' ) ) && $options['force_lang'] < 2 ) {
+			if ( ! pll_has_constant( 'PLL_CACHE_HOME_URL' ) && ( $options = get_option( 'polylang' ) ) && $options['force_lang'] < 2 ) {
```

Comment:
Would it be interesting to introduce a function like `pll_maybe_define_constant()`

Reply:
You mean like @Hug0-Drelon suggested [here](https://github.com/polylang/polylang/pull/1322#discussion_r1269506955)? 😅